ocaml/xapi/cli_frontend.ml

```diff
@@ -106,6 +106,15 @@ let rec cmdtable_data : (string*cmd_spec) list =
       flags=[Neverforward];
     };
 
+    "enable-getter-log",
```

Comment:
This command name does not follow the convention of starting with an object name of some sort. I see some existing commands that start with `log-`, so and that seems to fit here.

Also, having `enable` in the command name as well as in the argument is not great. Looking at the existing commands, it is more common to have a pair of `enable`/`disable` commands for this kind of thing.

More generally, isn't it better to do this using a config file? This is how logging is usually controlled.